composer.json

```diff
@@ -32,9 +32,15 @@
 		"php-stubs/wordpress-stubs": "^6.3",
 		"szepeviktor/phpstan-wordpress": "^1.3"
 	},
+	"repositories": [
+		{
+			"type": "vcs",
+			"url": "https://github.com/infinum/eightshift-libs.git"
+		}
+	],
 	"require": {
 		"php": "^7.4 || >=8.0",
-		"infinum/eightshift-libs": "^7.1.2"
+		"infinum/eightshift-libs": "dev-feature/optimizations"
```

Comment:
I recon this will be updated to v 8, right?

Reply:
correct once everything is finished with reviews and testing